tests/test_wsi_registration.py

```diff
@@ -1,7 +1,104 @@
+import pathlib
+
 import numpy as np
 import pytest
 
-from tiatoolbox.tools.registration.wsi_registration import match_histograms
+from tiatoolbox.tools.registration.wsi_registration import (
+    match_histograms,
+    prealignment,
+)
+from tiatoolbox.utils.misc import imread
+
+
+def test_prealignment(fixed_image, moving_image, fixed_mask, moving_mask):
+    """Test for prealignment of an image pair"""
+    fixed_img = imread(pathlib.Path(fixed_image))
+    moving_img = imread(pathlib.Path(moving_image))
+    fixed_mask = imread(pathlib.Path(fixed_mask))
+    moving_mask = imread(pathlib.Path(moving_mask))
```

Comment:
These fixtures should already be `pathlib.Path` objects according to the type hints. Therefore, it is unnecessary to wrap them in `pathlib.Path`. Additionally, this should probably be done in the function which is accepting the paths not here.
```suggestion
    fixed_img = imread(fixed_image)
    moving_img = imread(moving_image)
    fixed_mask = imread(fixed_mask)
    moving_mask = imread(moving_mask)
```